server.js

```diff
@@ -14,12 +14,22 @@ const config = require('./config.js')
 
 var useAutoStoreData = process.env.USE_AUTO_STORE_DATA || config.useAutoStoreData
 
-// TODO: Nunjucks config
-// var nunjucksConfig = {
-//   autoescape: true,
-//   noCache: true,
-//   watch: false // We are now setting this to `false` (it's by default false anyway) as having it set to `true` for production was making the tests hang
-// }
+
+var nunjucksConfig = {
+  autoescape: true,
+  noCache: true,
+  express: app,
+  watch: false // We are now setting this to `false` (it's by default false anyway) as having it set to `true` for production was making the tests hang
```

Comment:
Do you know why it was making them hang?

Reply:
It would appear that the lack of chokidar (dependency required for the watch feature) isn't installed & we don't currently see much value in using the feature (we don't mind manual restarts & the dependency would still be required in production if we enabled it